lib/index.js

```diff
@@ -63,13 +70,16 @@ module.exports = '${escape(text)}';
 `
 }
 
-function textToExport (text, path) {
-  return `module.exports['${escape(path)}'] = '${escape(text)}'`
+function textToExport (text, path, options) {
+  if (options.prefix) {
+    path = `${options.prefix}/${path}`
+  }
+  return `${options.global}['${escape(path)}'] = '${escape(text)}'`
 }
 
-function join (results) {
+function join (results, options) {
   return `'use strict';
-module.exports = Object.create(null);
+${options.global} = Object.create(null);
```

Comment:
These two functions (`textToExport` and `join`) are where the actual code change happens. The rest of the diff is to support and document the changes in these two functions.